CONTRIBUTING.md

```diff
@@ -137,22 +105,16 @@ There are also other test suites:
   on some utility functions in cabal-install you should run this test
   suite.
 
-* `cabal-install:solver-quickcheck` are QuickCheck tests on
-  cabal-install's dependency solver.  If you are working
-  on the solver you should run this test suite.
+* `cabal-install:long-tests` are QuickCheck tests on
+  cabal-install's dependency solver, VCS, and file monitoring code.
+  If you are working on the solver you should run this test suite.
 
 * `cabal-install:integration-tests2` are integration tests on some
   top-level API functions inside the `cabal-install` source code.
 
 For these test executables, `-p` which applies a regex filter to the test
 names.
 
-**Testing `cabal-install` Locally**
```

Comment:
I think this section should stay and just say "run cabal test". After all running tests locally is still encouraged!

Reply:
Same as above - calling `cabal test` is normal behavior, so i don't think it needs to be highlighted. We can debate this - I think it's unnecessary to point out details that people should _expect_ to already be a thing.